### ISSUE TITLE
Travis test fail

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#407]
Since 214235d the Travis test fail

**Comment by treerink** [comment]
The local nosetests go fine.
`````
UnsatisfiableError: The following specifications were found to be in conflict:
  - pyngl=1.5.0
  - python-eccodes=2.9.0
Use "conda search <package> --info" to see the dependencies for each package.
The command "conda env create -f environment.yml" failed and exited with 1 during .
`````

**Comment by treerink** [comment]
Solved with merge #408
